PaigeVannelli/shine
# Feature/new post form

## Pull Request
### Description
* Is this a feature or a fix?
Feature!

* Where should the reviewer start?
Please start at the types.ts document and the NewPostForm.tsx document.

* What functionalities do these changes affect? Why were these changes made? What was the motivation behind these changes?
Adds a functional form that takes in a title and content. On button press, the content is then saved locally and available for viewing on the home page.

* What outside features or research did you use to implement this fix?
Stack overflow.

* How should this be tested?
You can pull down the code and make sure that state is updating on the newPostForm page and on App after the submit button is pressed. Make sure that the correct pages are commented out on App under render

closes #20
